### PR TITLE
Optimized config loading and refactored Jmeter element custom exception logging

### PR DESCRIPTION
**Pull request by leelaprasadv**
Hey @QAInsights ,

This is a nice tool you have created there!
I just thought few parts of it could be refactored and some further optimized. 

So I made those changes in this PR...

- I've refactored the following:
  - The way config is being loaded. Instead of loading it multiple times, its better to load once and use it everywhere in the tool.  
  - Converted `if/elif/else` conditional checks to Dictionary based switch case for exception element logging to reduce number of condition checks.
- I've added the following:
  - `config.py` which will load the configuration at the start of application and holds the configuration for later use.
  - `exceptions.py` which is refactored if/else blocks from `engine.py`
- I've removed:
  - `f.close()` while loading jmx file in `app.py`, since file is opened with a context manager.

Cheer!
Prasad

### CODE REVIEW

**Comment by anthonygauthier** [comment]
Glad to see that I'm not the only one that saw great potential in the tool and started contributing 😄 ! Nice work @leelaprasadv !

File: core/engine.py [hunk 1]
```diff
@@ -57,87 +50,8 @@ def find_element_status(tree, element):
...
-            print_message(message_color=Colors.white, message="Consider adding Header Manager.")
-
-        elif element == 'TestAction':
-            print_message(message_color=Colors.red, message=f"{enabled_count} Test Action are enabled.")
-            print_message(message_color=Colors.white, message="Consider adding Test Action.")
-
-        else:
-            pass
+    # print custom logs for exception elements
+    Exceptions.check(element, flag, enabled_count, message)
```
Comment (anthonygauthier) [inline core/engine.py:54]: Much cleaner

**Comment by QAInsights** [comment]
Thanks @leelaprasadv and @delirius325 for the support :).

**Comment by QAInsights** [comment]
Thanks @leelaprasadv for the PR. It is merged now. 
@all-contributors please add @leelaprasadv  for code.

**Pull request merged**